packages/react-cosmos-loader/src/create-context.js

```diff
@@ -82,9 +84,13 @@ export function createContext(args: ContextArgs): ContextFunctions {
           updatedFixture = { ...fixture };
         }
 
+        if (!StateProxy) {
```

Comment:
I don't know why I did this lazy initialization in the past, but how about just doing `const StateProxy = createStateProxy({ updateInterval: 50 });` once, outside `createContext`?

Reply:
Yep